apps/coscrad-frontend/src/components/coscrad-byline/coscrad-byline.tsx

```diff
@@ -1,10 +1,31 @@
+import { Grid, Typography } from '@mui/material';
 import { COSCRADLogo } from '../coscrad-logo/coscrad-logo';
-import './coscrad-byline.css';
 
 export const COSCRADByline = (): JSX.Element => {
     return (
-        <span className="coscrad-byline">
-            A project built on the <COSCRADLogo /> platform.
-        </span>
+        <Grid
```

Comment:
I wonder if you could do a demo \ tutorial about using MUI Grid idiomatically so that we are all on the same page.

Reply:
Sure, we could look this over at standup